.github/workflows/tag.yaml

```diff
@@ -0,0 +1,25 @@
+name: create_tag
+on: [push]
+
+permissions:
+  contents: write
+
+jobs:
+  create_tag:
+    name: "Create tag"
+    if: contains(github.event.head_commit.message, 'Prepare')
```

Comment:
Is there something less surprising we can use to gate this logic?

If I were to submit a PR called "Prepare to be amazed", would that cause a tag?

Reply:
Let me see if gh event payload has some other properties that we can use for this verification. As far as I know, there wasn't much except to check the commit message this way to gate this logic. Since we would have different `release` name  to be different in the commit each time, we can't compare the whole string.

Reply:
Right, most importantly I don't want our release pipeline to be vulnerable to people submitting crafty commit messages.